docs/Internal Systems/Research & Findings/JekyllResearchTushar.md

```diff
@@ -0,0 +1,68 @@
+# Jekyll Research 
+
+_Jekyll is an open-source static site generator which is written in Ruby._ 
```

Comment:
I would probably change the file name to not include the author name as this documentation is for future teams, however you can detail the author in the file somewhere if you like.

Reply:
Same goes for the other files

Reply:
This change has been made all files now no longer contain names in the title.